core/loadpoint_plan.go

```diff
@@ -29,13 +29,17 @@ func (lp *Loadpoint) planRequiredDuration(maxPower float64) time.Duration {
 
 	if energy, ok := lp.remainingChargeEnergy(); ok {
 		requiredDuration = time.Duration(energy * 1e3 / maxPower * float64(time.Hour))
+
+		// Consider charge efficiency manually because we calulate outside the estimator
+		requiredDuration = time.Duration(float64(requiredDuration) / soc.ChargeEfficiency)
```

Comment:
Das würde ich nicht machen. wenn 20kWh verheizt werden würde ich genau das erwarten.

Reply:
Shit ich verstehe deinen Einwand leider nicht was meinst du? Ich habe die zeile nur von hinterm if else in das if gezogen. Was meins du genau? Das obere if hat sich im Ablauf nicht geändert. Eigentlich nur das ELSE.

Reply:
Stimmt:/ Ich würde bei fester Energiemenge erwarten, dass die auf dem Zähler landet, nicht zwingend im Auto. Ich ändere das entsprechend, war vorher schon odd.

Reply:
Ok ich bin gespannt was du änderst. Ich verstehe es nicht. Ich halte es für wichtig das es nicht im ELSE landet. Da sollte es jetzt richtig sein. Im weitesten sinne.

Reply:
Schau mal- ok?